components/form/LinkInverters.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import InverterGraphicIcon from '../icons/InverterGraphicIcon';
+import Link from 'next/link';
+import useMediaQuery from '~/lib/hooks/useMediaQuery';
+import Button from '../Button';
+import Spinner from '../Spinner';
+import { ChevronRightIcon } from '@heroicons/react/24/solid';
+
+const LinkInverters = () => {
+  const [showDisclaimer, setShowDisclaimer] = useState(false);
+  const mobile = useMediaQuery('(max-width: 768px)');
+
+  const mobileSkipButtonClass =
+    'w-full text-ocf-yellow nderline text-[14px] mt-[5px]';
+
+  const mobileInverterLinkClass =
+    'h-[54px] w-[308px] bg-ocf-yellow rounded-md font-semibold mb-[20px]';
+
+  const deslktopInverterLinkClass =
+    'h-[54px] w-[308px] text-ocf-yellow border-ocf-yellow border-[2px] rounded-md font-semibold';
+
+  return (
+    <div className="w-[320px] h-[550px] md:w-full flex flex-col items-center md:mt-[75px]">
+      <InverterGraphicIcon />
+      <div className="text-white mt-[30px] md:w-[485px] text-[20px] md:text-[24px]">
+        Would you like to link your inverter with Enode to provide better
+        forecasting?
+      </div>
+      {mobile && !showDisclaimer && (
+        <button
+          className="w-full text-right text-ocf-yellow underline text-[14px] mt-[5px]"
+          onClick={() => setShowDisclaimer(true)}
+        >
+          What&apos;s this?
+        </button>
+      )}
+      {(!mobile || showDisclaimer) && (
+        <div className="w-full text-ocf-gray-300 text-[14px] mt-[20px] md:w-[485px]">
+          Linking your inverter with Enode gives us access to your solar output
+          data, providing you with better forecasts and more information
+          available within our app.
+        </div>
+      )}
+
+      {mobile && showDisclaimer && (
+        <button
+          className="w-full text-right text-ocf-yellow underline text-[14px] mt-[5px]"
+          onClick={() => setShowDisclaimer(false)}
+        >
+          Show less
+        </button>
+      )}
+      <div className="flex flex-col md:justify-start justify-center md:mt-[50px] mt-auto">
+        <Link href="https://www.cnn.com/2023/04/17/us/kansas-city-teen-shot-wrong-house/index.html">
+          <button
+            className={
+              mobile ? mobileInverterLinkClass : deslktopInverterLinkClass
+            }
+          >
+            Yes, link my inverter
+          </button>
+        </Link>
+        {mobile && (
+          <button className={mobileSkipButtonClass}>Skip this step</button>
+        )}
+      </div>
+      <div className="hidden md:flex md:flex-row md:justify-end w-11/12 mx-auto mt-auto text-[5px]">
+        <Button disabled={false} variant="next-hover-button">
```

Comment:
if we make disabled default to false (since most buttons should not be disabled anyways), we can avoid having to pass in `disabled={false}`